requirements.txt

```diff
@@ -6,6 +6,3 @@ pyecma376-2>=0.2.4
 urllib3>=1.26,<2.0
 lxml-stubs~=0.5.1
 
-jsonschema~=4.7
-types-python-dateutil
-lxml-stubs~=0.5.1
```

Comment:
I'm still not sure about the dependencies, but ignore, if you're not done with them.